homeassistant/components/zabbix/__init__.py

```diff
@@ -29,6 +55,24 @@
                 vol.Optional(CONF_PATH, default=DEFAULT_PATH): cv.string,
                 vol.Optional(CONF_SSL, default=DEFAULT_SSL): cv.boolean,
                 vol.Optional(CONF_USERNAME): cv.string,
+                vol.Optional(CONF_PUBLISH_STATES_HOST, default=""): cv.string,
+                vol.Optional(CONF_RETRY_COUNT, default=0): cv.positive_int,
```

Comment:
Do we need to have a retry option? What's the behavior of zabbix seen for this?

Reply:
Nope. I implemented a default of 3